### DISCUSSION TITLE
Wiki Edit

### DISCUSSION THREAD

**Discussion by DevilsDesigns** [mueslimak3r/tv-intro-detection#11] [General]
Hey ive finished the wiki edit. All i need is the correct way for windows to find your jellyfin files do i need to do a certain command or how does it find the directory where your movies or tv shows are stored

**Comment by DevilsDesigns** [comment]
if you want to take a look so far:  [Wiki](https://github.com/DevilsDesigns/tv-intro-detection/wiki)

**Comment by mueslimak3r** [comment]
> Hey ive finished the wiki edit. All i need is the correct way for windows to find your jellyfin files do i need to do a certain command or how does it find the directory where your movies or tv shows are stored

maybe something like this:

**How the script finds your videos**

When using `jellyfin.py`, the script queries the jellyfin server and is sent a list of all the tv shows in your library. With that list of shows, it then queries the jellyfin server for the seasons and episodes for each of the shows. Included in the results of those queries is the location on the filesystem for each item (show, season, episode).

To account for jellyfin potentially using a different filesystem path to access the media than the system running the script would (eg: running in docker, running the script on a separate computer with the media drives mounted as network shares), `path mapping` is used to translate the path jellyfin sees into a path that the script can use.

When running `decode.py` on its own, the `-i` parameter is used to specify a folder that contains video files. `decode.py` doesn't do any searching beyond that single folder, so typically the provided folder will be a "season folder" that contains all the video files for that season of a show.

**Reply by DevilsDesigns** [reply]
Did you want me to add this to the wiki. Also do you want me to make a pull for the edits.

**Reply by mueslimak3r** [reply]
Yes to both

**Reply by DevilsDesigns** [reply]
i created a pull request let me know of any more edits or anything you want added or structered differently

**Reply by mueslimak3r** [reply]
https://github.com/DevilsDesigns/tv-intro-detection/pull/1

If this is the pull request you're talking about then you did it wrong. You need to open a PR that merges work done on your fork -> this repo.

**Reply by mueslimak3r** [reply]
I see that github docs can't be forked alongside the repo or merged upstream. Because of this I'm not going to use the github wiki feature. Instead I've set up a boilerplate github pages config in the /docs folder. Can you update your fork and add your changes to this?

It uses `just-the-docs` and the markdown reference is here: https://just-the-docs.github.io/just-the-docs/

For now I've just copied the contents of the readme but feel free to overwrite it
